.env.example

```diff
@@ -0,0 +1,3 @@
+CONFIG=local
```

Comment:
According to https://crates.io/crates/dotenv we can write `export` in front of these which can come in handy later.